src/camera.rs

```diff
@@ -18,23 +18,59 @@ fn spawn(mut commands: Commands) {
 
 pub fn update(
     keys: Res<ButtonInput<KeyCode>>,
-    mut query: Query<(Ref<OrthographicProjection>, &mut Transform), With<Camera2d>>,
+    mut query: Query<(&OrthographicProjection, &mut Transform), With<Camera2d>>,
```

Comment:
Just noting here that I previously used `Ref<OrthographicProjection>` and `is_changed` to ensure that the camera transform is not modified unless movement actually happens (or if the window is resized)

And with this PR, we just modify the camera transform unconditionally.

The original code was *probably* a micro-optimization, and resizing doesn't seem to cause any problems with, so I suppose this is OK.

Reply:
My understanding of the original code is that if the camera's projection changes, the camera's position will be re-clamped.
If the function keeps the same behavior, it may make the control flow more complicated.
So I wrote the restrictions directly at the end of the function.

Reply:
Yeah, that is the most important part 👍.